packages/plugin-snowplow/build.gradle

```diff
@@ -0,0 +1,87 @@
+plugins {
+    id 'org.jetbrains.kotlin.multiplatform'
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android-extensions'
+apply from: rootProject.file('gradle/publish-mpp.gradle')
+apply from: rootProject.file('gradle/android-library.gradle')
+
+kotlin {
+    android() {
+        publishLibraryVariants("release", "debug")
+        publishLibraryVariantsGroupedByFlavor = true
+    }
+
+    targets.all {
+        compilations.all {
+            kotlinOptions {
+                // jvmTarget = "1.8"
+                allWarningsAsErrors = false
+            }
+        }
+    }
+
+    sourceSets {
+        // COMMON
+        commonMain {
+            dependencies {
+                implementation kotlin('stdlib-jdk8')
+                implementation project(":packages:sdk-core")
+            }
+        }
+        commonTest {
+            dependencies {
+                implementation kotlin('test')
+                implementation kotlin('test-junit5')
+                implementation project(":packages:test-fixtures")
+            }
+        }
+
+        // ANDROID
+        androidMain {
+            dependsOn commonMain
+            dependencies {
+                implementation 'com.snowplowanalytics:snowplow-android-tracker:2.0.1@aar'
+            }
+        }
+        androidTest {
+            dependsOn commonTest
+        }
+    }
+
+    configure([targets["metadata"], android()]) {
+        mavenPublication {
+            groupId = project.group
+            version = project.version
+
+            artifact(jvmDocsJar)
+
+            pom {
+                name = 'Iteratively Snowplow Plugin'
+                description = 'Iteratively Analytics SDK for JVM'
+                url = 'https://iterative.ly'
+                licenses {
+                    license {
+                        name = 'The Apache License, Version 2.0'
+                        url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                    }
+                }
+                developers {
+                    developer {
+                        id = 'amplitude-dante-tam'
+                        name = 'Dante Tam'
+                        email = 'dante.tam@amplitude.com'
+                    }
+                }
+                scm {
+                    connection = 'scm:git:https://github.com/iterativelyhq/itly-sdk-jvm.git'
+                    developerConnection = 'scm:git:https://github.com/iterativelyhq/itly-sdk-jvm.git'
+                    url = 'https://github.com/iterativelyhq/itly-sdk-jvm'
+                }
+            }
+        }
+    }
+}
+
+apply from: rootProject.file('gradle/dokka-mpp-android-jvm.gradle')
```

Comment:
@dantetam I think this is causing your error. Switch it to `gradle/dokka-mpp-android.gradle`